### ISSUE TITLE
Transparency / Corruption Index

### ISSUE THREAD

**Issue by rgrp** [datasets/registry#57]
E.g. Transparency International index

@christophergandrud would you be interested in pull this one together? Also are there other political economy datasets that you think would be of general importance and interest (that would merit being "[core datasets](http://data.okfn.org/roadmap/core-datasets)")?

**Comment by Yannael** [comment]
I have just tried my first package for this one, see https://github.com/Yannael/corruption-perceptions-index

**Comment by rgrp** [comment]
@Yannael fantastic! Can you check the data package in the validator http://data.okfn.org/tools/validate and previewer http://data.okfn.org/tools/view and drop links to the results in here.

**Comment by Yannael** [comment]
It is now validated and view seems correct.

On Mon, Mar 2, 2015 at 3:20 AM, Rufus Pollock <notifications@github.com>
wrote:

> @Yannael <https://github.com/Yannael> fantastic! Can you check the data
> package in the validator http://data.okfn.org/tools/validate and
> previewer http://data.okfn.org/tools/view and drop links to the results
> in here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/datasets/registry/issues/57#issuecomment-76679872>.
>



-- 
=========================================
Yann-Aël Le Borgne
Machine Learning Group
Université Libre de Bruxelles

http://mlg.ulb.ac.be
http://www.ulb.ac.be/di/map/yleborgn
=========================================

**Comment by Yannael** [comment]
@rgrp I think this one is ready now

http://data.okfn.org/tools/validate?url=https%3A%2F%2Fgithub.com%2FYannael%2Fcorruption-perceptions-index

http://data.okfn.org/tools/view?url=https%3A%2F%2Fgithub.com%2FYannael%2Fcorruption-perceptions-index

**Comment by rgrp** [comment]
OK, great work. Some quick comments:

- View looks broken in that no data tables are displaying. I suspect looking at JS this is because having a '/" in a field name is bad news (sort of a bug in recline but anyway). Can we change Country/Territory either to simple Country or Jurisdiction or similar?
- I wonder if it would be useful to have all the data in one file. We could do this by adding a "year" column and then setting that for each source data year (are there any variations in data stored over the years? If so you would need to deal with that).
- datapackage.json seems to have some mis-indentation. Its not strictly invalid but would be nice to be consistently indented

**Comment by Yannael** [comment]
* View looks broken 
I had checked with Firefox and Chrome on MacOS X 10.7.5, there were no issues.  Checking now with Safari, I see that indeed nothing is displayed. I'll look into that, thanks.
* All data in one file: Yes there are variations (countries change from year to year, and besides, Transparency International did not use a consistent naming for the countries). I'll look deeper into that, indeed it would be nice to have a table with CPI/Years.
*  Datapackage.json: Noted

Thanks!

**Comment by Yannael** [comment]
@rgrp I merged all the tables in one, and updated package files accordingly.
Two questions related to datapackage.json fields:
* The license is unclear, so I mentioned that in the License section. In this case, is it better to leave the license field of datapackage.json empty, or can I put the PDDL license there?
* Is it good practice to include contributors/maintainers fields?

**Comment by rgrp** [comment]
@Yannael re license i would go for PDDL but have a note in the License section about data source and potential rights. See this thread for more https://discuss.okfn.org/t/copyright-on-data-sources/189 (Aside: we really should get guidance on this into the FAQ at http://data.okfn.org/doc/publish-faq as it comes up a lot)

**Comment by Yannael** [comment]
@rgrp Thanks. I sent a mail to Transparency International to make sure what the copyright is.

**Comment by rgrp** [comment]
@Yannael let's get this in and not wait on response from TI. We can put in a "reservation" in the README. Personally given size of dataset I doubt there are DB rights but we can just comment to that affect.

**Comment by Yannael** [comment]
@rgrp 
ok, I modified the README, and added to the datasets repository, and to core-list and catalog-list
